### DISCUSSION TITLE
Can an external robot script be put into the source code and run like an instruction？

### DISCUSSION THREAD

**Discussion by FIZxingchen** [ORelio/Minecraft-Console-Client#2636] [General]
_I_ was interested in this because I had written a script that I felt was too cumbersome to put out and wanted to run it in EXE like an MCC instruction

**Comment by Nekiplay** [comment]
See this examples
https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config/ChatBots

**Reply by FIZxingchen** [reply]
When I put my own external script in C:UsersAdministratorDesktopmcccodeMinecraftClientconfigChatBots I get an error, for example:
Severity Code Description Project File Line Suppression status
Error CS0103 Name "MCC" MinecraftClient C:UsersAdministratorDesktopmcccodeMinecraftClientconfigChatBotstell.cs 3 activity does not exist in the current context

**Comment by Nekiplay** [comment]
And methods and events in https://github.com/MCCTeam/Minecraft-Console-Client/blob/master/MinecraftClient/Scripting/ChatBot.cs
